src/logic/notifications/notificationTypes.ts

```diff
@@ -65,7 +66,10 @@ export const NOTIFICATIONS: Record<NotificationId, Notification> = {
     message: 'Error connecting to your wallet',
     options: { variant: ERROR, persist: true },
   },
-
+  COULDNT_FETCH_SIGNATURES_ERROR_MSG: {
+    message: 'Couldn’t fetch all signatures for this transaction',
```

Comment:
Could you please change it to: `Couldn’t fetch all signatures for this transaction. Please reload page and try again.` - makes it more actionable.

Reply:
Great, regarding your previous comment, you mean to show a message in the modal saying that the signatures could't be fetched?

Reply:
> Great, regarding your previous comment, you mean to show a message in the modal saying that the signatures could't be fetched?

No, I was thinking instead of showing the modal at all, already show the error notification. My reasoning: The user has 0 chance to fix the problem in the modal, since the signatures are missing -> why even display it? What do you think?

Reply:
Yeah sure! I'll do that

Reply:
@tschubotz i was able to show the error before the modal. But if the user clicks on execute in the first seconds of opening the screen, the transaction is undefined(i guess the transaction is being loaded asyncronously) so i can't verify if there are enough confirmations. What should we do, let the user open the modal an show the error after the confirm in the modal, or it should't happen that the modal could be opened even if the transaction is undefined.

Reply:
> What should we do, let the user open the modal an show the error after the confirm in the modal, or it should't happen that the modal could be opened even if the transaction is undefined.

yea, ideally I'd say the modal shouldn't open but it should rather wait until the data has been loaded.
however, let be pragmatic. if the above is too tedious to achieve, I think it's fine if the error notification gets shown
- before opening the modal when the info about confirmations is present
- before triggering metamask when the info wasn't present early enough

what do you think?

Reply:
Sure, i'll do that.